wallet/src/money.rs

```diff
@@ -18,6 +18,57 @@ use tuxedo_core::{
     verifier::Sr25519Signature,
 };
 
+use std::{thread::sleep, time::Duration};
+
+/// The default number of coins to be minted.
+const DEFAULT_NUM_OF_COINS: u128 = 100;
+
+/// The Max value of coins can be minted.
+const MAX_NUM_OF_COINS: u128 = 1000;
```

Comment:
I don't see any value in limiting the number you can mint. Minting would not be exposed in a production network anyway.